dom.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="dom.css" />
+    <title>JavaScript DOM Manipulation</title>
+    <style>
+      #experienceForm {
+        display: none;
+        flex-direction: column;
+        margin-top: 10px;
+      }
+      #experienceForm label,
+      #experienceForm textarea,
+      #experienceForm input,
+      #experienceForm button {
+        margin-bottom: 10px;
+        display: flex;
+        flex-direction: column;
+      }
+      #experienceForm label[for="companyName"],
+      #experienceForm input[type="text"],
+      #experienceForm label[for="startDate"],
+      #experienceForm input[type="date"],
+      #experienceForm label[for="endDate"],
+      #experienceForm input[type="date"] {
+        flex-direction: row;
+        align-items: center;
+      }
+      .experience-entry {
+        margin: 10px 0;
+        padding: 10px;
+        border: 1px solid #ccc;
+        background-color: #f0f0f0;
+      }
+    </style>
+    <header>
+      <!-- The Header section -->
+      <div class="header">
+        <a href="signup.html">Login!</a>
+      </div>
+    </header>
```

Comment:
It should be login.html since text is login not signup.

Reply:
fixed!